src/core.api.js

```diff
@@ -87,6 +88,16 @@ class OctotreeService {
     }
   }
 
+  async _loadAllTree(context, repo) {
```

Comment:
The name doesn't make sense. It is called "loadX" but it actually returns boolean. Maybe 'shouldLoadEntireTreeAtOnce'. Also, don't pass in 'context', just pass in 'adapter'.